core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -713,12 +713,7 @@ public static boolean getRecacheStudyAfterUpdate() {
     }
     
     public static String getDbVersion() {
-        String version = properties.getProperty(DB_VERSION);
-        if (version == null)
-        {
-            return "0";
-        }
-        return version;
+        return DATABASE_VERSION;
```

Comment:
This is the other part of that. Currently the system reads the version from the DB, and another version from some runtime properties file, and checks they are the same. Which is pointless, as nothing is connected to the code at all. The intent here was to propose moving one of the versions into source, so it's versioned. This then can be checked against the version read from the DB, which is actually done in `DaoInfo`. It's horrible, I admit, because the name here is misleading. Yes, let's change it as you suggest, as it's not too widely referenced.